### ISSUE TITLE
Issue linking libudev.so.1

### ISSUE THREAD

**Issue by etu** [sergev/dmrconfig#8]
When I made the nixos package I used the git commit a4c5f893d2749727493427320c7f01768966ba51 as the revision we build on.

Then I tried to update to e4c3c3f92054656d9b03fc80e432b65425c35885 (latest at the moment).

When I build the old version, an `ldd result/bin/dmrconfig` on the binary I get this output:
```
	linux-vdso.so.1 (0x00007ffcac761000)
	libusb-1.0.so.0 => /nix/store/wcj72npwrfpsh078wrxm5d2va7di4g36-libusb-1.0.22/lib/libusb-1.0.so.0 (0x00007febeaf42000)
	libc.so.6 => /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/libc.so.6 (0x00007febeab8e000)
	libudev.so.1 => /nix/store/bir61x9w8y6shssys65qwzysqfalrwp1-systemd-239-lib/lib/libudev.so.1 (0x00007febea968000)
	libgcc_s.so.1 => /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/libgcc_s.so.1 (0x00007febea752000)
	libpthread.so.0 => /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/libpthread.so.0 (0x00007febea533000)
	/nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/ld-linux-x86-64.so.2 => /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib64/ld-linux-x86-64.so.2 (0x00007febeb15c000)
	librt.so.1 => /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/librt.so.1 (0x00007febea32b000)
	libmount.so.1 => /nix/store/q0kgnq21j0l2yd77gdlld371246cwghh-util-linux-2.32.1/lib/libmount.so.1 (0x00007febea0d2000)
	libblkid.so.1 => /nix/store/q0kgnq21j0l2yd77gdlld371246cwghh-util-linux-2.32.1/lib/libblkid.so.1 (0x00007febe9e81000)
	libuuid.so.1 => /nix/store/q0kgnq21j0l2yd77gdlld371246cwghh-util-linux-2.32.1/lib/libuuid.so.1 (0x00007febe9c79000)
```

If I do it on the newer one:
```
	linux-vdso.so.1 (0x00007ffe66fd5000)
	libusb-1.0.so.0 => /nix/store/wcj72npwrfpsh078wrxm5d2va7di4g36-libusb-1.0.22/lib/libusb-1.0.so.0 (0x00007f7683585000)
	libc.so.6 => /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/libc.so.6 (0x00007f76831d1000)
	libudev.so.1 => not found
	libudev.so.1 => /nix/store/bir61x9w8y6shssys65qwzysqfalrwp1-systemd-239-lib/lib/libudev.so.1 (0x00007f7682fab000)
	libgcc_s.so.1 => /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/libgcc_s.so.1 (0x00007f7682d95000)
	libpthread.so.0 => /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/libpthread.so.0 (0x00007f7682b76000)
	/nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/ld-linux-x86-64.so.2 => /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib64/ld-linux-x86-64.so.2 (0x00007f768379f000)
	librt.so.1 => /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/librt.so.1 (0x00007f768296e000)
	libmount.so.1 => /nix/store/q0kgnq21j0l2yd77gdlld371246cwghh-util-linux-2.32.1/lib/libmount.so.1 (0x00007f7682715000)
	libblkid.so.1 => /nix/store/q0kgnq21j0l2yd77gdlld371246cwghh-util-linux-2.32.1/lib/libblkid.so.1 (0x00007f76824c4000)
	libuuid.so.1 => /nix/store/q0kgnq21j0l2yd77gdlld371246cwghh-util-linux-2.32.1/lib/libuuid.so.1 (0x00007f76822bc000)
```

And if I try to execute the new one. I get the following error:
```
./result/bin/dmrconfig: error while loading shared libraries: libudev.so.1: cannot open shared object file: No such file or directory
```

**Comment by sergev** [comment]
Thank you for bug report. Newer version adds a file serial.c, which needs udev explicitly. I've modified Makefile to add -ludev to $(LIBS). Please try this version: 4924d00283c3c81a4b8251669e42aecd96b6145a

**Comment by etu** [comment]
@sergev That works! Thanks :+1:

**Comment by sergev** [comment]
Fixed in 4924d00283c3c81a4b8251669e42aecd96b6145a.
